README.md

```diff
@@ -0,0 +1,47 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HTML & CSS PROJECT 3 Newsweek-Bootstrap-clone
+
+> Clone of THE NEWSWEEK page using Bootstrap
+
+<img src="img/ScreenShot-desktop.png" width="700">
+<img src="img/ScreenShot-tablet.png" width="600">
+<img src="img/ScreenShot-mablet.png" width="500">
```

Comment:
- [ ] Please fix this image, it's not loading properly in the README file.
Having broken images makes the README file, and by extension the project, look less professional.